tests/Oryx.Integration.Tests/LocalDockerTests/NodeEndToEndTests.cs

```diff
@@ -91,6 +91,63 @@ public async Task CanBuildAndRunNodeApp_UsingZippedNodeModules()
                 });
         }
 
+        [Fact]
+        public async Task CanBuildAndRunNodeApp_UsingZippedNodeModules_WithoutExtracting()
```

Comment:
How is this going to work? Should there be mounting of the zip using a zip driver or something?

Reply:
My original plan was to let the app fail due to lack of node_modules, but given our framework I decided to manually extract it to the app folder